stable/filebeat/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: A Helm chart to collect Kubernetes logs with filebeat
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: filebeat
-version: 0.3.0
-appVersion: 6.2.3
+version: 0.3.1
+appVersion: 6.3.0
```

Comment:
Is increasing the image version really a bugfix?
Increasing the minor version of the appVersion and image seems like cause for increasing at least the minor version of the chart.

Reply:
@rendhalver  good catch!
fixed.